sites/hashdev/src/components/MdxPageContent.tsx

```diff
@@ -10,6 +11,27 @@ type MdxPageContentProps = {
 export const MdxPageContent: FunctionComponent<MdxPageContentProps> = ({
   serializedPage,
 }) => {
-  // @ts-expect-error @todo fix this
-  return <MDXRemote {...serializedPage} components={mdxComponents} />;
+  return (
+    <Box
```

Comment:
Did you consider doing this in `MuiTypographyThemeOptions.ts`?

Reply:
I did not, because I think this CSS rule needs to be applied to the parent of the typography element - not the typography element itself. Is there a way this could be accomplished in `MuiTypographyThemeOptions.ts`?

Reply:
For reference I'm using the ["Adjacent sibling combinator" CSS rule](https://developer.mozilla.org/en-US/docs/Web/CSS/Adjacent_sibling_combinator)

Reply:
Adding a rule of `{ marginTop: 0 }` with a selector like `&[class*="MuiTypography-hashHeading"] + [class*="MuiTypography-hashHeading"]` inside the `root` object in `MuiTypographyThemeOptions` ought to work – try that?

Reply:
Interesting, will try that - thanks :)

Reply:
Thanks Nate, that was a good idea and is working for me locally. Have addressed in https://github.com/hashintel/hash/pull/1167/commits/730027746314b7183fb885bd6df428511bd8f10d